### DISCUSSION TITLE
a/yt-dlp-usage

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#20] [Announcements]
# a/yt-dlp-usage

https://blog.yasking.org/a/yt-dlp-usage.html

**Comment by Tomp-Akoe** [comment]
今天用了一下，国内的一些网站好了还是不太好用，比如新片场，爱优腾等

**Reply by sincerefly** [reply]
没太好的办法，爬虫类型的工具需要持续开发和维护，用爱发电不容易

**Reply by Tomp-Akoe** [reply]
好的好的，深刻理解，辛苦了！

**Comment by vgshots-io** [comment]
感谢博主！
